kafka_consumer/datadog_checks/kafka_consumer/client/kafka_python_client.py

```diff
@@ -30,9 +30,9 @@ def token(self):
 
 
 class KafkaPythonClient(KafkaClient):
-    def __init__(self, check, config) -> None:
+    def __init__(self, check, kafka_config) -> None:
         self.check = check
-        self.config = config
```

Comment:
I ended up using `self.kafka_config` instead of `self.config` here since `self.config` is used by the [ConfigMixin](https://datadoghq.dev/integrations-core/meta/config-models/#interface), but this is only initialized right before the first check run. I wanted to make sure that I avoid any collisions in `self.config`.